src/components/CurrentPlan/CurrentPlan.js

```diff
@@ -130,7 +130,10 @@ const CurrentPlan = ({
             case 'S':
               price = subItem.nextPaymentPrice;
               currency = subItem.nextPaymentCurrency;
-              renewalDate = dateFormat(subItem.expiresAt);
+              renewalDate =
+                subItem.expiresAt === 2145913200
+                  ? 'the next season start'
+                  : dateFormat(subItem.expiresAt);
```

Comment:
Not sure if it was a part of the ticket but we will probably need more changed for this, eg:
Unsubscribe l:165, 375
Resubscribe l:76, 102
SwitchPlanPopup, l:177 (+ various other lines)
+ to review if there are other places